models/olids/base/base_olids_schedule_practitioner.sql

```
@@ -6,27 +6,29 @@
     src.lds_versioner_event_id,
-    src.record_owner_organisation_code,
-    src.lds_datetime_data_acquired,
-    src.lds_initial_data_received_date,
+    src.lds_datetime_first_acquired,
+    src.lds_datetime_update_acquired,
     src.lds_is_deleted,
-    src.lds_start_date_time,
+    src.lds_start_datetime,
     src.lds_lakehouse_date_processed,
     src.lds_lakehouse_datetime_updated
 FROM {{ source('olids_common', 'SCHEDULE_PRACTITIONER') }} src
```

SQLFluff (4.2.1)

[error] Implicit/explicit aliasing of table.

(AL01)